src/data/map.js

```diff
@@ -533,9 +533,9 @@ const getPokestops = async (minLat, maxLat, minLon, maxLon, updated = 0, showPok
         }
 
         if (excludedItems.length === 0) {
-            excludeItemSQL = 'OR (quest_item_id IS NOT NULL)';
+            excludeItemSQL = 'OR (quest_reward_type IS NOT NULL AND quest_reward_type = 3 AND quest_item_id IS NOT NULL)';
         } else {
-            let sqlExcludeCreate = 'OR ((quest_item_id IS NOT NULL AND quest_item_id NOT IN (';
+            let sqlExcludeCreate = 'OR (quest_reward_type IS NOT NULL AND quest_reward_type = 3 AND quest_item_id IS NOT NULL AND quest_item_id NOT IN (';
             for (let i = 0; i < excludedItems.length; i++) {
                 if (i === excludedItems.length - 1) {
                     sqlExcludeCreate += '?))';
```

Comment:
```js
sqlExcludeCreate += '?)';
```
Need to remove this extra `)`

Reply:
I think that's fine: `OR (... NOT IN (...))`?

Reply:
The ending ) is on line 553
```
OR (quest_reward_type IS NOT NULL AND quest_reward_type = 3 AND quest_item_id IS NOT NULL AND quest_item_id NOT IN (?, ?, ?, ?)))
```

Reply:
Okay @versx is adding more stuff even after finalizing `sqlExcludeCreate`. 🤷